pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,16 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+// backoff is the underlying backoff struct we use to prevent thundering herd drama, esp when this runs in a controller.
+var backoff = &URLBackoff{Backoff: util.NewBackOff(1*time.Second, 120*time.Second)}
```

Comment:
For completeness, here are the details, 

timothysc [4:18 PM] 
so you want to just backoff a single client then, without global visibility?
@lavalamp: ^ It's fine by me I guess.
Isn't technically accurate for several error conditions but they will eventually backoff too

lavalamp [4:36 PM] 
@timothysc: Right. I think they'll all back off eventually, and if that's not good enough we can fix it in a second pass. But I'd want to see some sort of evidence that it's necessary first.

timothysc [4:43 PM] 
Sgtm @jayunit100 FYI..

- Just remove global visibility and only backoff per client is the consensus.